<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerProduct</class>
 <widget class="QWidget" name="registerProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>REGISTRO DE PRODUCTOS</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Titillium Web</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Titillium Web</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Unidades:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_agregar">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AGREGAR</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>Producto:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txt_unidades"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txt_precio"/>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_previous">
        <property name="font">
         <font>
          <family>Segoe UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_next">
        <property name="font">
         <font>
          <family>Segoe UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txt_producto"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_new">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NUEVO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
